jobs/bodhi-trigger.Jenkinsfile

```diff
@@ -115,7 +115,18 @@ if (matching_streams.size() == 1) {
     error("multiple non-devel matching streams found: ${matching_streams}")
 }
 
-currentBuild.description = msg.update.builds[0].nvr
+// Find the builds in the update we care about and update the
+// description with those build NVRs.
```

Comment:
I guess it's a bit more nuanced than that because the Bodhi update might have other packages that are in FCOS but not explicitly listed in `bodhi-testing.yaml`. It'd be confusing if we then fail because of that package and not the visible package in the build description. Agree it's a corner-case though.

Reply:
honestly I think we should be triggering on all FCOS SRPMS.

I think it would have caught something like this https://github.com/coreos/fedora-coreos-tracker/issues/1876, which, yeah, we don't have gating so it still would have got in, but at least we know about it sooner. i.e. just getting a build built without even running tests is useful.